Source/Source.php

```diff
@@ -141,7 +141,7 @@ function menu_main() {
 		$t_menu_options = array();
 
 		if ( plugin_config_get( 'show_repo_link' ) ) {
-			$t_page = plugin_page( 'index', false, 'Source' );
+			$t_page = plugin_page( 'index', true, 'Source' );
```

Comment:
This makes the URL returned by plugin_page() a full URI with protocol, host, etc.
I do not understand why it is necessary to change this in the context of allowing CJK chars, and furthermore AFAICT it works just fine with the current value (short URL with only the path).